large-message-core/src/main/java/com/bakdata/kafka/AbstractLargeMessageConfig.java

```diff
@@ -169,6 +171,10 @@ public class AbstractLargeMessageConfig extends AbstractConfig {
                     + "e.g. for EKS `/var/run/secrets/eks.amazonaws.com/serviceaccount/token`.";
     public static final String S3_JWT_PATH_CONFIG_DEFAULT = "";
     public static final String S3_SECRET_KEY_DEFAULT = "";
+    public static final String S3_REQUEST_CHECKSUM_CALCULATION_DOC =
+            "AWS request checksum validation mode to use when uploading to S3. Leave empty to use the AWS SDK default.";
+    public static final String S3_REQUEST_CHECKSUM_CALCULATION_DEFAULT =
```

Comment:
I think we should be able to use null as a default. Then we don't need to customize anything on the client builder and it is more robust if the default changes